Storage/src/Connection/Rest.php

```diff
@@ -628,6 +629,53 @@ private function sendWithRetry($resource, $method, $args)
             $args
         );
 
+        $retryIdentifierHash = Uuid::uuid4()->toString();
```

Comment:
The hash represents requests which won't be retried as well. Should we name it to something like `$requestHash`?

Reply:
Yes, sounds better.

Did the change [af0e43c](https://github.com/googleapis/google-cloud-php/pull/5741/commits/af0e43c40fb3d06f1eb1649a437cf04305ab765a)